core/src/main/scala/org/apache/spark/internal/config/Deploy.scala

```diff
@@ -19,48 +19,59 @@ package org.apache.spark.internal.config
 
 private[spark] object Deploy {
   val RECOVERY_MODE = ConfigBuilder("spark.deploy.recoveryMode")
+    .version("0.8.1") // No JIRA ID, commit ID: d66c01f2b6defb3db6c1be99523b734a4d960532
```

Comment:
Ah, it's okay to don't comment in the code. I meant to leave a Github PR comment for reviewers :-) .. thanks for leaving the traces.

Reply:
OK. I have deleted it.